### PR TITLE
QOL update

### PR DESCRIPTION
**Pull request by alxnsk**
Hi, amazing work you have done! Played entire game using this system.
I decided to commit a few hacks and tweaks that I've done for personal use, maybe you find them useful.

First, I automated the process of leveling up identity.
Label "Experience" is now a button. You can click on it, and it will roll "1d10 / 2 rounded up" adding the result to existing percents.
image

Second, I have seen the bug when, after you reset failing notches to zero, they're resetting themselves back after some time, or page refresh. There are two solutions to this.

1) Replacing the button with a handlebar element that will do the job. And this is what I did.
image

2) Using the actual ".update" function, which will update all states in all places. I guess we shouldn't modify states directly. FoundryVTT has its own DB of some sort, and it commits local changes to it somehow, sometimes. Need to actually read docs to understand, but it looks like it replaces the new "0" value with an old value after the page refresh.

In short, not like this:
`this.actor.system.shockGauge[$(event.currentTarget).data("shock-meter")].notches.failed = 0;`

But something like this:
```
let entry = await this.object.items.get(identityID);
entry.update({
    "system.percentage": finalPercent,
    "system.hasExperience" : false
});
```

Just an example, of course. I haven't read docs myself XD

And I have added Russian translation, and I needed to modify some parameters in CSS sheet, to properly fit translated text.
I don't know if you need it, but I did it, so it's here.

### CODE REVIEW

**Comment by patrickburk1988** [comment]
Hey, thanks for using the system and taking the time to work on it!

I actually had the failed notch bugfix and an identity improvement button in the works already, so I've gone ahead and pushed those changes. My implementation is different, though, so your personal changes should work as well, I think?

Full disclosure: I still don't 100% understand Git, and I'm not sure how to partially resolve a pull request. 😬 So I just pushed your Russian translation on my own and credited you in the patch notes. I did change a few of the strings and attempted to update the translation accordingly, but I don't speak Russian, so I'm not optimistic I got it right.
